[IDE] Error when adding checkstyle_ide.xml configuration

While following the [instruction ](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#checkstyle) to configure the checkstyle plugin for IntelliJ, the following error was thrown on Step 11 (`Click "Next", then "Finish"`):

```
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'scope' does not exist, please check the documentation
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:473)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
	at org.infernus.idea.checkstyle.service.cmd.OpCreateChecker.execute(OpCreateChecker.java:61)
	at org.infernus.idea.checkstyle.service.cmd.OpCreateChecker.execute(OpCreateChecker.java:26)
	at org.infernus.idea.checkstyle.service.CheckstyleActionsImpl.executeCommand(CheckstyleActionsImpl.java:130)
	at org.infernus.idea.checkstyle.service.CheckstyleActionsImpl.createChecker(CheckstyleActionsImpl.java:60)
	at org.infernus.idea.checkstyle.service.CheckstyleActionsImpl.createChecker(CheckstyleActionsImpl.java:51)
	at org.infernus.idea.checkstyle.checker.CheckerFactoryWorker.run(CheckerFactoryWorker.java:46)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module JavadocMethod - Property 'scope' does not exist, please check the documentation
	at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:128)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:468)
	... 7 more
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Property 'scope' does not exist, please check the documentation
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.tryCopyProperty(AutomaticBean.java:227)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:194)
	at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:123)
	... 9 more
```

The checkstyle version is selected to be `8.42` (latest)
IntelliJ is 2021.1.1 (build number IU-211.7142.45)

--- a/buildSrc/src/main/resources/checkstyle_ide_fragment.xml
+++ b/buildSrc/src/main/resources/checkstyle_ide_fragment.xml
@@ -37,6 +37,6 @@
     <!-- Public methods must have JavaDoc -->
     <module name="JavadocMethod">
         <property name="severity" value="warning"/>
-        <property name="scope" value="public"/>
+        <property name="accessModifiers" value="public"/>
     </module>
 </module>

--- a/buildSrc/version.properties
+++ b/buildSrc/version.properties
@@ -4,7 +4,7 @@ lucene            = 8.9.0-snapshot-efdc43fee18
 bundled_jdk_vendor = adoptopenjdk
 bundled_jdk = 16.0.1+9
 
-checkstyle = 8.39
+checkstyle = 8.42
 
 # optional dependencies
 spatial4j         = 0.7

